recipes/pysptools/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "pysptools" %}
+{% set version = "0.14.2" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "3d95c8b823c8f0fbe143428b3b3edbe6261890ef7f067f11e728c9061cb859a4" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed . 
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - scipy
+    - scikit-learn >=0.18
+    - spectral >=0.17
+    - matplotlib
+
+test:
+  imports:
+    - pysptools
+    - pysptools.abundance_maps
+    - pysptools.classification
+    - pysptools.detection
+    - pysptools.distance
+    - pysptools.eea
+    - pysptools.material_count
+    - pysptools.noise
+    - pysptools.sigproc
+    - pysptools.skl
+    - pysptools.spectro
+    - pysptools.tests
+    - pysptools.util
+
+about:
+  home: 'https://pysptools.sourceforge.io/'
+  license: Apache 2.0
+  license_family: APACHE
+  license_file: 'pysptools/LICENSE.txt'
+  summary: A hyperspectral imaging tools box
+  description: PySptools is a hyperspectral and spectral imaging library that provides spectral algorithms for the Python programming language. Specializations of the library are the endmembers extraction, unmixing process, supervised classification, target detection, noise reduction, convex hull removal and features extraction at spectrum level. For more information, see the docs.
```

Comment:
Could you split this up over multiple lines?

Reply:
Thanx! Ok, done.